nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -460,7 +456,7 @@ def inference_step(self, dataloader_iter, batch_idx, mode, dataloader_idx=0):
 
     def inference_epoch_end(self, outputs, mode, data_cfg):
         # Parent class will handle logging of the loss.
-        if not outputs:
+        if not outputs or not outputs[0]:
```

Comment:
This seems risky as well. Can we ask PTL for an API for outputs? It seems we never know what we're getting with it.